CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ Write the date in place of the "Unreleased" in the case a new version is release
 
 ## v0.1.0-b31 (Unreleased)
 
+### Changed
+
+- Refactored internal server function ``get_root_tree()`` to not use FastAPI
```

Comment:
It's still using the dependency injection, it's just not using the overrides.

Reply:
I just copied the doc string from the last version- looks like the CI just needs linting run again and for us to find a good wording here